README.md

```diff
@@ -46,73 +71,57 @@ This will output similar to the following:
-
-```yaml
-GOOGLE_APPLICATION_CREDENTIALS: "no-file"
-```
+Some tasks within the frontend are run in their own dedicated microservice. To read more about why some aspects are moved to a microservice and how they behave read [here](./microservices/README.md).
 
-This is a special value that will cause the caching service to automatically ignore any cache requests, and force request through to the backend service. Allowing to fail gracefully.
+### Social Image Cards
 
-### Running In Dev Mode
+Social Image Cards allow the creation of dynamic Package Cards to be displayed, either when linked to directly, or when a link to a packages page on the frontend has been shared to services that display the image websites prompt to.
 
-Using `npm run start:dev` you can run the Package Frontend Server in Development mode. This does two notable things:
-  - Disables Remote Cache Features Natively. Setting `GOOGLE_APPLICATION_CREDENTIALS: "no-file"` is not necessary when run in dev mode.
+To learn more about Social Image Cards, how they behave, and how to contribute take a look at the [Social Image Cards Readme](./microservices/social-cards/README.md).
 
-### Creating a New Theme
+### Download Link
 
-Creating a new theme should be rather simple.
+The Download Link allows easy linking to our most recent Cirrus CI or 'alpha' builds of Pulsar. This lets the worry of finding the relevant link be left to this microservice rather than on contributors.
 
-First create the theme itself, by laying out the theme variables in `./src/site.css` the top of this file should have every other supported theme listed along with the variables that can and need to be modified for your theme.
+To learn more about the Download Link Microservice take a look at it's [Readme](./microservices/download/README.md).
 
-Once the theme is created, ensure to add it as an option to set in `./public/site.js` within the function `changeTheme` add a new check in the switch statement. Setting the body to have your theme as an attribute value, and saving the new theme option into the users local storage.
+Additionally for easy access on how to use the download link take a look at the [Download Link Table](/docs/download_links.md).
 
-Lastly we need to ensure users are able to choose this theme when they'd like to. Within `./ejs-views/partials/header.ejs` add your new theme as a button towards the bottom of the page, along with all other buttons. Ensure your theme name is passed to the `changeTheme` function.
+### Webhooks
 
-Now with your theme created double check that the same name is used *exactly* in each location.
-  - The EJS Template
-  - The Client Side JavaScript
-  - The Client Side CSS
+The Webhooks Microservice exists as a translation tool from GitHub Sponsors Webhooks to Discord. Since this type of webhook is not natively supported.
```

Comment:
```suggestion
The Webhooks Microservice exists as a translation tool from the GitHub Sponsors Webhooks to our Discord server since this type of webhook is not natively supported by Discord.
```